Disable -flto as it is currently doesn't work with -Wl,--as-needed

Index: cairo/build/configure.ac.warnings
===================================================================
--- cairo.orig/build/configure.ac.warnings	2012-02-02 01:48:14.000000000 +0100
+++ cairo/build/configure.ac.warnings	2012-03-27 10:46:00.851499771 +0200
@@ -29,7 +29,7 @@
 
 dnl We also abuse the warning-flag facility to enable other compiler
 dnl options.  Namely, the following:
-MAYBE_WARN="$MAYBE_WARN -fno-strict-aliasing -fno-common -flto"
+MAYBE_WARN="$MAYBE_WARN -fno-strict-aliasing -fno-common"
 
 dnl Also to turn various gcc/glibc-specific preprocessor checks
 MAYBE_WARN="$MAYBE_WARN -Wp,-D_FORTIFY_SOURCE=2"
